l10n-ms/src/main/java/com/example/l10nadmin/controller/L10nController.java

```diff
@@ -23,14 +21,47 @@
 public class L10nController {
 
 	private final L10nService l10nService;
+	private final L10nRepository l10nRepository;
```

Comment:
It's against best practice to use the repository directly in the controller (unless it's Spring Data REST). You've wrapped the other 2 methods in L10nService, wrap this one too.